content/guides/references/configuration.md

```diff
@@ -204,11 +204,12 @@ creating `e2e` and `component` objects inside your Cypress configuration.
+| Option               | Default                               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| -------------------- | ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `setupNodeEvents`    | `null`                                | Function in which node events can be registered and config can be modified. Takes the place of the (deprecated) plugins file. [Please read the notes for examples on using this.](#setupNodeEvents)                                                                                                                                                                                                                                             |
+| `supportFile`        | `cypress/support/index.js`            | Path to file to load before test files load. This file is compiled and bundled. (Pass `false` to disable)                                                                                                                                                                                                                                                                                                                                       |
+| `specPattern`        | `cypress/e2e/**/*.cy.{js,jsx,ts,tsx}` | A String or Array of glob patterns of the test files to load.                                                                                                                                                                                                                                                                                                                                                                                   |
+| `excludeSpecPattern` | `*.hot-update.js`                     | A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses `minimatch` with the options: `{dot: true, matchBase: true}`. We suggest using [https://globster.xyz](https://globster.xyz) to test what files would match. <br><br> The **/node_modules/** pattern is automatically added to `specExcludePattern`, and does not need to be specified (and can't be overridden). |
```

Comment:
Instead of putting all this content here, can you move the details to an "excludeSpecPattern" section in the [notes](https://deploy-preview-4349--cypress-docs.netlify.app/guides/references/configuration#Notes) and just have "A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests." along with a link to that new section here, like what's done for `setupNodeEvents`? 

The same goes for the `component` option.